Client/Frontend/Popup/PaymentHandlerPopupView.swift

```diff
@@ -0,0 +1,284 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import BraveShared
+import Shared
+import Lottie
+
+class PaymentHandlerPopupView: PopupView {
+    fileprivate var dialogImage: UIView?
+    fileprivate var titleLabel: UILabel!
```

Comment:
Can we make all of these `private` instead of `fileprivate`?
Can we also make them all `let` instead of `var and finally, can we move the `optionals`? I don't think they need forced optional typing.

They all seem to be initialized in the constructor.

Usually we do:
```swift
messageLabel = UILabel().then {
    $0.font = ....
}
```

for example.